src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java

```diff
@@ -281,7 +281,7 @@ private static void fireRocket(final PlayerID player, final Territory attackedTe
           final int[] rolls = bridge.getRandom(highestMaxDice, numberOfAttacks, player, DiceType.BOMBING,
               "Rocket fired by " + player.getName() + " at " + attacked.getName());
           for (int i = 0; i < rolls.length; i++) {
-            final int r = rolls[i] + highestBonus;
+            final int r = Math.max(-1, rolls[i] + highestBonus);
```

Comment:
Thanks for adding the comments in `StrategicBombingRaidBattle`; I wouldn't have realized the -1 minimum value was due to a 0-based roll vs. a 1-based roll.  If they're appropriate to the two lines in `RocketsFireHelper`, as well, could you please add them in a future PR?

Reply:
Yeah, its pretty funky that way the code handles the dice rolls and I didn't notice til a few folks further tested it. I didn't add the comments to RocketsFireHelper as there already is a comment on dice being 0 based right below and it does the +1 calculation right there so much more obvious.

Reply:
Ahh, I just didn't expand the diff high enough.  I see the comment you're talking about. :+1: